### ISSUE TITLE
[Question] How to use PAGE_URL with RegEx?

### ISSUE THREAD

**Issue by Smorvel** [Pitmairen/selection-search#129]
`https://alitools.io/en/showcase?search={%PAGE_URL|replace:/(\d+)/$1/g}` - is not working

**Expected Effect**
- Replace this
`https://www.aliexpress.com/item/4000563108664.html`
- to this
`4000563108664`
- and get
`https://alitools.io/en/showcase?search=4000563108664`

**Comment by Pitmairen** [comment]
This is not possible at the moment, the replace converter only works with the selection placeholders.

**Comment by Smorvel** [comment]
@Pitmairen Thanks for the answer =) So it's a suggestion then.

**Comment by Pitmairen** [comment]
Yes, it makes sense for it to work with the other variables too.
